web3/tools/pytest_ethereum/deployer.py

```diff
@@ -1,8 +1,8 @@
 from typing import (
     Any,
     Callable,
-    Dict,
 )
+from typing import Dict  # noqa: F401
```

Comment:
Was there a limit your for dict? Weird that it imports typing twice

Reply:
Not sure. I put the `noqa` line after it in the original single import and something in linting split it out. Maybe isort?

Reply:
I think you can say: `self.strategies: Dict[str, Callable[[Package], Package]] = {}` on line 30 below and get rid of this weird import stuff